### ISSUE TITLE
How to install and run meteor-aREST-mqtt ?

### ISSUE THREAD

**Issue by tuteming** [marcoschwartz/meteor-aREST-mqtt#1]
How to install and run meteor-aREST-mqtt ?

also how to build a own "cloud.arest.io" ?

thanks

tutm

**Comment by marcoschwartz** [comment]
I'll come up with a decent tutorial in the future, but at the moment the only thing I can say is that you just need to install Meteor from https://www.meteor.com/ and start the project :)

**Comment by tuteming** [comment]
Tkanks
2015年11月30日 上午12:57於 "marcoschwartz" notifications@github.com寫道：

**Comment by tomasmcm** [comment]
I'm also not able to run this project on meteor. I'm getting a similar error to this user: http://stackoverflow.com/questions/21385334/meteorify-a-cloned-project
I believe we need the `.meteor` folder to be able to run this locally

**Comment by marcoschwartz** [comment]
Indeed, that's something I need to do in the future. I still need to work on separating things specific to the live server (at cloud.arest.io) before I can release this :)

**Comment by jallen2281** [comment]
For anyone setting this up on linux :

download and unzip the package:
wget https://github.com/marcoschwartz/meteor-aREST-mqtt/archive/master.zip
unzip master.zip

Assuming you have node/npm &MongoDB installed already 
First install meteor

curl https://install.meteor.com/ | sh

Create a new meteor project, copy your files over & switch to the directory
meteor create cloud
cd meteor-aREST-mqtt-master/
mv \* ../cloud
cd ../cloud

proceed installing deps:

npm install iron --save
npm install iron-meteor --save
npm install mosca --save
meteor add iron:router
meteor add percolate:synced-cron
meteor add meteorhacks:npm
meteor add mkarliner:mosca

Run the app, and a setup process will occur and then prompt you to run it a second time.
meteor
=>please run your app again

meteor
=> App running at: http://localhost:3000/

Cheers!
 ~Jared

**Comment by tuteming** [comment]
Thanks, I have been install on a Rp2 by similar method.
2016年3月12日 上午12:29於 "jallen2281" notifications@github.com寫道：

**Comment by wblaircox** [comment]
Anyone have issues with clients then connecting to the MQTT broker?

**Comment by wblaircox** [comment]
Hi Marco,
I've gone over this all weekend. I've copied everything and followed the instructions I paid for in your book. I can go to my server IP:3000 and see line output. Is that telling me the broker is actually working? I've tried client tools that will show a basic connection to cloud.arest.io, but doing the same to my server IP results in cannot connect issue. That would appear to me that the broker is running, but not accepting client connections?

**Comment by marcoschwartz** [comment]
@wblaircox Sorry to hear you are still having issues here. Can you reach to me at contact@openhomeautomation.net? Then I'll help you directly on your issue. Thanks!

**Comment by sudheer1994** [comment]
@wblaircox I think you forgot to open port 1883 for mosca.

**Comment by sudheer1994** [comment]
Hi Marco,
I have done everything you mentioned in tutorial and able to run your arest but problem is i am not able to control the connected esp8266 module. Please Help.

**Comment by smartman2012** [comment]
Hi Marco
I still have some problems to build my server on ubuntu 14.04 VPS server. If I build my server based on your book everything goes fine at the end mup.js is different than what you mentioned and I don't know how to configure it. if go based on what @jallen2281 write here I have an issue again. 
please write a step by step installation for whom don't know much about meteor or other things you work with.
thank you
